docs/pubsub-usage.rst

```diff
@@ -213,13 +228,13 @@ Fetch pending messages for a pull subscription:
 
 .. doctest::
 
-   >>> from gcloud.pubsub import Topic
-   >>> from gcloud.pubsub import Subscription
-   >>> topic = Topic('topic_name')
-   >>> subscription = Subscription('subscription_name', topic)
-   >>> with topic:
-   ...     topic.publish('this is the first message_payload')
-   ...     topic.publish('this is the second message_payload',
+   >>> from gcloud import pubsub
+   >>> client = pubsub.Client()
+   >>> topic = client.topic('topic_name')
+   >>> subscription = pubsub.Subscription('subscription_name', topic)
+   >>> with topic.batch() as batch:
+   ...     batch.publish('this is the first message_payload')
+   ...     batch.publish('this is the second message_payload',
    ...                   attr1='value1', attr2='value2')
    >>> received = subscription.pull()  # API request
```

Comment:
I think this might be a bigger thing (aka, I should create a separate issue), but wouldn't we want to get a list (or iterator) of objects back, which has all the stuff we're interested in, rather than a tuple of the `(ackId, message)` ?

That is, I think it might be nicer to write:

```python
messages = subscription.pull()
for message in messages:
  print message.getAckId()  # ReceivedMessage.ackId (string)
  print message['attr1']  # PubsubMessage.attributes (dict)
  print message.getData()  # PubsubMessage.data (bytes)
  print message.getId()  # corresponds to PubsubMessage.messageId (string)
```